.github/workflows/release.yml

```diff
@@ -34,13 +41,32 @@ jobs:
           source-url: https://nuget.pkg.github.com/open-feature/index.json
 
       - name: Install dependencies
-        if: ${{ steps.release.outputs.releases_created }}
         run: dotnet restore
 
       - name: Pack
-        if: ${{ steps.release.outputs.releases_created }}
         run: dotnet pack --no-restore
 
       - name: Publish to Nuget
-        if: ${{ steps.release.outputs.releases_created }}
         run: dotnet nuget push "src/**/*.nupkg" --api-key "${{ secrets.NUGET_TOKEN }}" --source https://api.nuget.org/v3/index.json
+
+  sbom:
+    runs-on: ubuntu-latest
+    needs: release-please
+    if: ${{ needs.release-please.outputs.release_created }}
```

Comment:
It may be worth enabling [continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error) until we've seen this job run successfully.

Reply:
It should not block the release even if it fails, but it's a good point regardless to keep the noise down in case of a failure. I'll add your suggestion.

Reply:
I've added `continue-on-error` now 😃